sstables/sstables_manager.cc

```diff
@@ -62,8 +62,7 @@ sstable_writer_config sstables_manager::configure_writer(sstring origin) const {
     cfg.summary_byte_cost = summary_byte_cost(_db_config.sstable_summary_ratio());
 
     cfg.correctly_serialize_non_compound_range_tombstones = true;
-    cfg.correctly_serialize_static_compact_in_mc =
-            bool(_features.cluster_supports_correct_static_compact_in_mc());
+    cfg.correctly_serialize_static_compact_in_mc = true;
     cfg.origin = std::move(origin);
```

Comment:
Please follow up with removal of the member variable, which can no longer vary.

Reply:
Yeah, I left it as is for now, because some tests validated what happens if the variable is false. But it's actually better to purge the variable right now (as well as the one above, left from the previous series), I'll do it in v2.

Reply:
Looks like you forgot this.

Reply:
oh, right. In v3 then